### PR TITLE
Main merge group

### PR DESCRIPTION
**Pull request by ahillary**
* [ ] file structure changes
* [ ] everyone to the same local main

### CODE REVIEW

File: script/seed.js [hunk 1]
```diff
@@ -224,10 +224,65 @@ const orders = async () => {
...
+      quantity: 10,
+    },
+  });
+  let thing = product[1];
+
+  console.log('user1', one);
+  console.log('product1', thing);
+
+  let end = await one.addProducts([thing]);
+  console.log('end', end);
```
Comment (saba-mo) [inline script/seed.js]: please update

File: server/db/models/index.js [hunk 2]
```diff
@@ -3,8 +3,10 @@ const Product = require('./product');
 const Order = require('./order');
 const Cart = require('./cart');
 
-Product.belongsToMany(Order, {through: Cart, foreignKey: 'product_id'});
-Order.belongsToMany(Product, {through: Cart, foreignKey: 'order_id'});
+// user swapped for order for association testing purposes because we have a db seeded with users and products.
+
+Product.belongsToMany(User, {through: Cart, foreignKey: 'product_id'});
+User.belongsToMany(Product, {through: Cart, foreignKey: 'order_id'});
```
Comment (saba-mo) [inline server/db/models/index.js]: revert

File: server/index.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+const path = require('path');
+const express = require('express');
+const morgan = require('morgan');
+const compression = require('compression');
+const session = require('express-session');
+const passport = require('passport');
+const SequelizeStore = require('connect-session-sequelize')(session.Store);
+const db = require('./db');
+const sessionStore = new SequelizeStore({db});
+const PORT = process.env.PORT || 8081;
```
Comment (saba-mo) [inline server/index.js]: change to 8080

**Review by saba-mo — CHANGES_REQUESTED** [review]
see comments

**Comment by ahillary** [comment]
Updates complete

**Review by saba-mo — APPROVED** [review]

**Pull request merged**